service-loadbalancer/haproxy_reload

```diff
@@ -23,6 +23,4 @@
 # -s soft reload, wait for pids to finish handling requests
 # -f send pids a resume signal if reload of new config fails
 
-socat /tmp/haproxy - <<< "show servers state" > /var/state/haproxy/global
```

Comment:
Is that an haproxy bug? or do they document a min reloads per second? can we just limit the reloads per second to work around this?